sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/E2E.Tests/AzureMonitorTraceExporterLiveTests.cs

```diff
@@ -0,0 +1,110 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Monitor.OpenTelemetry.Exporter.E2E.Tests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Diagnostics;
+    using System.Linq;
+    using System.Threading.Tasks;
+
+    using Azure.Core.TestFramework;
+    using Azure.Monitor.OpenTelemetry.Exporter;
+    using Azure.Monitor.OpenTelemetry.Exporter.E2E.Tests.TestFramework;
+    using Azure.Monitor.Query;
+    using Azure.Monitor.Query.Models;
+    using global::OpenTelemetry;
+    using global::OpenTelemetry.Resources;
+    using global::OpenTelemetry.Trace;
+
+    using NUnit.Framework;
+
+    /// <summary>
+    /// Collection of tests to evaluate the <see cref="AzureMonitorLogExporter"/>.
+    /// </summary>
+    public class AzureMonitorTraceExporterLiveTests : RecordedTestBase<MonitorExporterTestEnvironment>
+    {
+        public AzureMonitorTraceExporterLiveTests(bool isAsync) : base(isAsync) { }
+
+        [RecordedTest]
+        [PlaybackOnly("Add field level validations")]
+        public async Task VerifyTraceExporter()
+        {
+            // SET UP
+            ActivitySource src = new ActivitySource("TestActivitySource");
+            string roleName = nameof(VerifyTraceExporter);
+
+            var resourceAttributes = new Dictionary<string, object>
+            {
+                { "service.name", roleName },
+            };
+
+            var resourceBuilder = ResourceBuilder.CreateDefault().AddAttributes(resourceAttributes);
+
+            using var tracerProvider = Sdk.CreateTracerProviderBuilder()
+                .SetResourceBuilder(resourceBuilder)
+                .AddSource("TestActivitySource")
+                .SetSampler(new AlwaysOnSampler())
```

Comment:
I think the sampler can be removed?
Correct me if I'm wrong, but no sampler would be equivalent to AlwaysOn.